CONTRIBUTORS.txt

```diff
@@ -1,5 +1,6 @@
 - Contributors -
 ----------------
+2-5
```

Comment:
I think we prefer actual name.

Reply:
Removed, I don't actually want attribution